src/component/Calculator.js

```diff
@@ -1,35 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import CalculatorShow from './calculatorShow';
 import CalculatorBtn from './calculatorBtn';
 import calculate from '../logic/calculate.js';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.eventHandler = this.eventHandler.bind(this);
-    this.state = {
-      total: 0,
-      next: 0,
-      operation: '',
-    };
-  }
+function Calculator() {
```

Comment:
- [ ] Please use `arrow functions` instead of the `function` keyword, if you want to know more about why you should do this, take a look at the comment above 😄